packages/core/src/index.ts

```diff
@@ -11,4 +11,4 @@ export {
   loadAllTranslations,
   loadTranslation,
 } from './load-translations';
-export type { TranslationFile } from '@vocab/types';
+export * from './types';
```

Comment:
Should be an `export type` with all the types listed explicitly, or does it not matter in this case?

Reply:
`export type` was [introduced in TypeScript 5.0](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#support-for-export-type). It would fail for projects on <5.0. In fact, that's why I made this commit [`44806cd` (#137)](https://github.com/seek-oss/vocab/pull/137/commits/44806cdc3012bc629557f58c1fe697b345c75801), because VS Code was fine, but `pnpm lint` failed because it was using v4.9.5.

Reply:
Oh, I just meant `export type { /* all the types here */ } from './types'`. Sucks that `export type *` was only supported very recently.

Reply:
In that case, it doesn't matter. In the output `.d.ts` file `export X` and `export type X`  work the same way.